mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.tsx

```diff
@@ -18,25 +24,55 @@ export const AboutProjectPage = () => {
 
   return (
     <div className={styles.pageWrapper}>
-      <Title
-        level={HeadingLevel.h2}
-        text={LangService.aboutProject.mainTitle[language]}
-      />
+      <div className={styles.pageContainer}>
+        <div className={styles.aboutBlock}>
+          <Title
+            level={HeadingLevel.h2}
+            text={LangService.aboutProject.mainTitle[language]}
+            className={styles.title}
+          />
+          <Title
+            level={HeadingLevel.h3}
+            text={LangService.aboutProject.mainSubTitle[language]}
+            className={styles.title}
+          />
+          <div className={styles.aboutDescription}>
+            <div className={styles.aboutTextBlock}>
+              <div>
+                {LangService.aboutProject.descriptionSubTitle[language]}
+              </div>
+              <Title
+                level={HeadingLevel.h3}
+                text={LangService.aboutProject.descriptionTitle[language]}
+              />
+              <div>
+                {renderMarkdown(
+                  LangService.aboutProject.descriptionList[language],
+                )}
+              </div>
+            </div>
+            <Image
+              src={theme === Theme.DARK ? logoLight : logo}
```

Comment:
Do we use the same trick in the header? If yes - let's create specific component for such purpose)

Reply:
created Logotype component